drivers/modem/Kconfig.quectel-bg9x

```diff
@@ -54,6 +54,14 @@ config MODEM_QUECTEL_BG9X_PASSWORD
 	  context.  This value is specific to the network provider and may
 	  need to be changed.
 
+config MODEM_QUECTEL_BG9X_UNSOL_RDY
+	string "Unsolicited response string expected from the modem"
+	default "RDY"
+	help
+	  This string is used to determine when modem powered and ready
+	  to accept AT commands. If this value is not matching the modem
+	  response, the init will fail with timeout.
+
```

Comment:
Why does this need to be a kconfig value? This should never be changing. The AT command responses are defined and do not change.

Reply:
it does not have to be, but otherwise there has to be a differentiation between MODEM_QUECTEL_BG9X and presumably MODEM_QUECTEL_BG9x_BG95. I don't know how else this could be accomplished?

Reply:
![image](https://user-images.githubusercontent.com/7346660/233668578-8d198298-b3a4-47f2-8080-38f508facfaf.png)
Like I've mentioned in the original ticket, as "the issue". (screenshot from `Quectel_BG95BG77BG600L_Series_AT_Commands_Manual_V2.0-2.pdf`)